src/sections/player.ts

```diff
@@ -68,6 +68,8 @@ export class Player extends LitElement {
       let override = overrides.find(
         (value) =>
           (media_title && media_title === value.mediaTitleEquals) ||
+          (media_artist && media_artist === value.mediaArtistEquals) ||
```

Comment:
looks like these are undefined? media_artist and media_album_name?
Did you actually try it?

Reply:
I did, but when I was trying to fix the mess of multiple pull requests I re-did the branch and I inadvertently forgot to copy one line.
I updated it.